src/dropbox-api.js

```diff
@@ -166,7 +166,7 @@ module.exports = {
 		};
 		Object.assign(clientSession, {
 				generateAuthUrl: (input) => {
-					return `${OAUTH2_AUTHORIZE}?client_id=${config.client_id}&response_type=code&redirect_uri=${config.redirect_uri}`;
+					return `${OAUTH2_AUTHORIZE}?client_id=${config.client_id}&response_type=code&redirect_uri=${config.redirect_uri}&state=${config.state}`;
```

Comment:
Unfortunately, by default it will be: `&state=undefined`.
I recommend adding default value for `state` in `config` object as well as mentioning this extension in README.md. 
Thank you for your PR.

Reply:
You are of course correct, I changed it so state is only added if it is given and added information about it in the README.